test/integration/generated/polymorphicSkipNormalize/src/generated/mediaServicesClient.ts

```diff
@@ -1,18 +1,28 @@
 import * as coreClient from "@azure/core-client";
+import * as coreAuth from "@azure/core-auth";
 import { MediaServicesClientOptionalParams } from "./models";
 
 export class MediaServicesClient extends coreClient.ServiceClient {
   /**
    * Initializes a new instance of the MediaServicesClient class.
+   * @param credentials Subscription credentials which uniquely identify client subscription.
    * @param options The parameter options
    */
-  constructor(options?: MediaServicesClientOptionalParams) {
+  constructor(
+    credentials: coreAuth.TokenCredential,
```

Comment:
I found this change is because the swagger has such definition https://github.com/Azure/azure-rest-api-specs/blob/a57e89f8c3e1215201694cc0d6e9a51842573df0/specification/mediaservices/data-plane/LiveVideoAnalytics.Edge/preview/2.0.0/LiveVideoAnalyticsSdkDefinitions.json#L17-L23

Reply:
even if we have set security: undefined, the m4 seems still generate the security.

Reply:
Due to https://github.com/Azure/autorest.typescript/issues/1373 we currently don't have the testmodeler dependency so we should be able to have it in a separate PR